k/kernel-headers-system-docker.yml

```diff
@@ -0,0 +1,9 @@
+kernel-headers-system-docker:
```

Comment:
Why 2 services? (one for docker, one for system-docker) 

I'd just leave the system-docker one and just call it `kernel-headers` 
(without the "-system-docker" suffix).